app/models/submission.rb

```diff
@@ -6,15 +6,15 @@ class Submission < ActiveRecord::Base
   belongs_to :course
 
   belongs_to :exercise,
-    -> (submission) {
-      if submission.respond_to?(:course_id)
-        # Used when doing submission.exercise
-        where(course: submission.course)
-      else
-        # Used when doing submissions.include(:exercises)
-        Exercise.joins(:submission)
-      end
-    }, foreign_key: :exercise_name, primary_key: :name
+             lambda do |submission|
+               if submission.respond_to?(:course_id)
+                 # Used when doing submission.exercise
+                 where(course: submission.course)
+               else
+                 # Used when doing submissions.include(:exercises)
+                 Exercise.joins(:submission)
+               end
+             end, foreign_key: :exercise_name, primary_key: :name
```

Comment:
I think the old indentation was more appropriate here but it's ok to leave this if the tool can't easily be made to understand lambdas as a special case.